admin/services/check-form.service.js

```diff
@@ -176,6 +176,19 @@ const checkFormService = {
       }
     })
     return canDelete
+  },
+  /**
+   * Build a form name based on the file name.
+   * @param fileName
```

Comment:
This param returns a string, undefined, and false.  Whilst it would be better to have a single return type, at least please document the return types in use.